prow/cluster/arm/kubernetes_external_secrets.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: kubernetes-client.io/v1
```

Comment:
I think we actually don't need any of these. Since its only for the release job which arm never will. I can see any argument that they should be the same, but its easier to add secrets than remove them (since we never know blast radius), so I would be in favor of moving all of these. I believe we don't need any at all.

I'd keep the external secrets deployment though... not sure why, it just feels right :-). We may have some secrets we need eventually